## Suture Simulation Configuration
"link7_radius": 0.00175
"link8_radius": 0.00175
"link9_radius": 0.00125
"head_name": "suture_head"
"end_name": "suture_end"
"element_name_prefix": "s"

"realization_iteration_times": 1
"continuity_iteration_times": 5

"sampling_time_sec": 0.001 
"gravity": 1 

"eta_d": 200
"vfi_threshold": 2
"self_collision_check_threshold": 5
"bed_safe_distance": 0.0

"collision_check": true
"self_collision_check": false    # do not use
"penetration_check": false
"penetration_check_threshold": 0.001

"vrep_ip": "192.168.133.1"  # the ip address of the computer on which the Vrep (the simulator you want to operate) is runnning.
"vrep_port": 20001


//0.618033988